src/Typography/Typography.component.tsx

```diff
@@ -84,7 +84,13 @@ export interface TypographyProps extends SpaceProps, React.HTMLAttributes<any> {
     lineHeight?: number | string;
 }
 
-const StyledTypography = (tag: TypographyTags) => styled(tag)<TypographyProps>`
+export interface StyledTypographyProps extends TypographyProps {
+    $color?: 'inherit' | 'initial' | string;
```

Comment:
Added `$color` here as well to keep it out of the dom

Reply:
I really don't like the idea of embedding a special character into the props. Where did you derive this from?

Reply:
This is a pattern I used in `Button` component for `fontSize`, `height`, and `size`. I based it off of the discussion in this thread https://github.com/styled-components/styled-components/pull/2093 (which afaict, still doesn't have a definite resolution). It's merely internal to the component, so the consumer still just uses the `color` prop when using `Typography`.

Reply:
I feel like `fontColor` is a bad idea because it's substandard, but that this creates confusion in a code-case where observables may be present. Since there's no clear winner here, I defer to your judgement. While I think the `$` is likely to raise more questions than a substandard name like `fontColor`, `$` is less technically misleading as it doesn't create a intermediate standard between the DOM and the component props. ¯\_(ツ)_/¯